app/controllers/courses_controller.rb

```diff
@@ -1,2 +1,72 @@
 class CoursesController < ApplicationController
+  before_action :set_course, only: [:show, :edit, :destroy]
+  before_action :authorized, only: [:create, :edit, :destroy]
+  # GET /todos
+  def index
+    # @courses = Course.all
+    json_response(@current_user.courses)
+  end
+
+  def show
+    @my_course = UserCourse.where(user_id: current_user.id, course_id: @course.id)
+    if !@my_course.empty?
+      json_response(@my_course.map(&:course))
+    else
+      json_response(
```

Comment:
I think the status code for this should be 400 (an invalid request) but you are returning a 200